eng/targets/Helix.Common.props

```diff
@@ -32,7 +32,7 @@
     <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Debian.9.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="(Fedora.28.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-28-helix-09ca40b-20190508143249" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="(Fedora.33.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-helix-20210120000908-a9df267" Platform="Linux" />
```

Comment:
This queue name also appears in src/Testing/src/HelixQueues.cs (though that `const` is unused) and src/Tools/dotnet-watch/test/GlobbingAppTests.cs. Both should change before this is merged.

Reply:
Thanks I'll update globbing apps, are those constants used for anything other than the RunHelix script?  My preference would be to nuke that file now as it seems like yet another case of something that will get stale easily if its meant to mimic the helix matrix queues

Reply:
I'm fine w/ removal if the `const`s no longer provide value.

Reply:
Cool pretty sure it was a leftover vestige of andrew's flaky attribute stuff, I don't think it would work with the docker random commit things anyways